チュートリアルで外税表示の税額が受け取れない

チュートリアルを試しました。
外税方式で表示するテストが失敗してしまいます。
理由は、taxが0を受け取っている。
内税方式で表示するテストは成功します。

```
> jinzo-ningen-test@1.0.0 test C:\bin\jinzo-ningen-test
> jest

  console.warn node_modules/jinzo-ningen/lib/login.js:18
    kintone login successful

 FAIL  ./index.test.js (8.237s)
  計算テスト
    入力と保存、表示
      新規作成画面での入力操作
        × 消費税額と合計額が外税で正しく計算できたこと (386ms)
        √ 消費税額と合計額が内税で正しく計算できたこと (315ms)

  ● 計算テスト › 入力と保存、表示 › 新規作成画面での入力操作 › 消費税額と合計額が外税で正しく計算できたこと

    expect(received).toEqual(expected) // deep equality

    Expected: 792
    Received: 0

      40 |         const total = await jz.getNumber(page, 'total');
      41 |
    > 42 |         expect(tax).toEqual(792);
         |                     ^
      43 |         expect(total).toEqual(9900);
      44 |       });
      45 |

      at Object.toEqual (index.test.js:42:21)

Test Suites: 1 failed, 1 total
Tests:       1 failed, 1 passed, 2 total
Snapshots:   0 total
Time:        8.518s, estimated 9s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```


環境
windows 10 pro
node v8.14.0
npm 6.9.0

jest-puppeteer.config.js
では
`    ignoreDefaultArgs: ['--disable-extensions'],`
を追加している（chromeにポリシー適用されているため拡張が入ってしまうを防ぐ）

試したこと

- calc.jsを
`      event.record.tax.value = 1251;`
のように固定値にしても、外税方式は0を受け取り、内税方式は1251を受け取りエラーになる。
常にtaxの額を受け取れないわけではない様子。
- `        await jz.waitForMoment(page);`
をいくつか増やしてみる。
 (2485ms)のように時間はかかっているが、依然0を受け取ってエラーなる。
- ヘッドレスモードにしてみるが変化なし。

--- a/lib/page-utils.js
+++ b/lib/page-utils.js
@@ -235,14 +235,15 @@ const typeInputTexts = async (page, inputSelector, fieldCode, value) => {
 
 const selectRadioButton = async (page, fieldCode, item) => {
   const fieldId = await getFieldIdFromCode(page, fieldCode)
-  await page.evaluate(
+  const itemId = await page.evaluate(
     ({ fieldId, item }) => {
       const nodes = document.querySelectorAll(`.field-${fieldId} .input-radio-item-cybozu input`)
-      const node = Array.from(nodes).find(node => node.value === item)
-      node && node.click()
+      const selectedItem = Array.from(nodes).find(node => node.value === item)
+      return selectedItem && selectedItem.id
     },
     { fieldId, item }
   )
+  if (itemId) await page.click(`[id="${itemId}"`)
 }
 
 const selectDropdown = async (page, fieldCode, item) => {
@@ -251,14 +252,14 @@ const selectDropdown = async (page, fieldCode, item) => {
   const fieldId = await getFieldIdFromCode(page, fieldCode)
   await page.click(`.field-${fieldId} .gaia-argoui-select`)
   // 表示された項目から選択したい項目を検索してそれをクリック
-  const dropdownItemId = await page.evaluate(selectItem => {
+  const itemId = await page.evaluate(selectItem => {
     const items = document.querySelectorAll('ul.gaia-argoui-selectmenu:not([style*="display: none"]) .goog-option')
     const selectedItem = Array.from(items).find(item => {
       return item.querySelector('.goog-menuitem-content').textContent === selectItem
     })
-    return selectedItem.id
+    return selectedItem && selectedItem.id
   }, item)
-  await page.click(`[id="${dropdownItemId}"]`)
+  if (itemId) await page.click(`[id="${itemId}"]`)
 }
 
 const checkCheckBox = async (page, fieldCode, itemCheckMap) => {

